src/interfaces/wallet.h

```diff
@@ -195,6 +196,13 @@ class Wallet
         bool& in_mempool,
         int& num_blocks) = 0;
 
+    //! Fill PSBT
+    virtual TransactionError fillPSBT(PartiallySignedTransaction& psbtx,
+        bool& complete,
+        int sighash_type = 1 /* SIGHASH_ALL */,
```

Comment:
2bc10122874421ba7ce379325ebc2420955b1b9b

Looks like default arguments aren't used in these interfaces, @ryanofsky coincidence or is there a reason for that?

Reply:
Maybe for now move the defaults for wallet.cpp and just have `psbtc` and `complete` arguments for now.

Reply:
Or even just make this `unsignedFilllPSBT(psbtx)` which `assert(complete == false)` and makes the GUI code simpler?

Reply:
I think refactoring the interface can be done some other time. External signers #16546 will set `sign` to true, and the result may or may not be complete (e.g. multisig).